.travis.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         - (cd / && go get github.com/mattn/goveralls)
       script:
         - make test-sanity test-unit test-markdown
-        - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage.out -repotoken=$COVERALLS_TOKEN
+        # - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage.out -repotoken=$COVERALLS_TOKEN
```

Comment:
But we cannot do it in this pr commit. See that the issue faced in your Travis will not be faced here. 
You can add the ENV VAR TOKEN with an coveralls.io token to solve it in your Travis. 

OR 

Create a branch from it just to do the tests and then comment on this line.